exchanges/stream/websocket.go

```diff
@@ -65,8 +70,8 @@ func (w *Websocket) Setup(s *WebsocketSetup) error {
 
 	w.features = s.Features
 
-	if w.features.Subscribe && s.Subscriber == nil {
-		return errors.New("features have been set yet channel subscriber is not set")
+	if s.Subscriber == nil {
+		return errSubscriberUnset
```

Comment:
This one is out of scope 😄 When making sure all websockets had subscribers, GateIO does not have it as a feature under `WebsocketCapabilities: protocol.Features{`. Would you be able to add it?